src/vue/components/morph.vue

```diff
@@ -269,8 +269,6 @@ export default {
   },
   computed: {
     lexemes () {
-      // A call to `defDataReady` will force this computed prop to recalculate every time definitions data is updated
-      let defs = this.$store.getters['app/defDataReady']
```

Comment:
are you sure we don't need this anymore (using the app/shortDefsDataReady property instead? It is possible for short defs to get updated after the initial population of the morphology data.

Reply:
I believe in order to be taken into consideration `defs` has to be a part of the returned expression's result. `defs` here is unused, and I think Vue.js is smart enough to to count it in. But I'm not 100% sure, so I'll better return it back for safety.

Reply:
ok, thanks.